### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JayrajSingh65**
## 🛠️ Fixes Issue
Updated readme. Md

## 👨‍💻 Changes proposed

Readme
.md

## ✔️ Check List (Check all the applicable boxes) 

- 
- [x] The title of my pull request is a short description of the requested changes.

## 📷 Screenshots &#11015;&#65039
[

Screenshot 2022-04-24 164245
-------

Screenshot 2022-04-24 164318

### CODE REVIEW

**Comment by JayrajSingh65** [comment]
Waiting for you to merge this ----

**Comment by Pradumnasaraf** [comment]
> Waiting for you to merge this ----



> Waiting for you to merge this ----

Hey Jayraj, thank you for your patience, @FrancescoXX will soon review it and take further steps

**Comment by Pradumnasaraf** [comment]
> Waiting for you to merge this ----



> Waiting for you to merge this ----

Hey Jayraj, thank you for your patience, @FrancescoXX will soon review it and take further steps

**Comment by ewainy** [comment]
Hi @JayrajSingh65 great work gathering these, thank you for this! I think we might need to format it a little to be consistent with the rest of the README.md. I would like to see the YouTube list capitalized.

Also with emojis, are we having emoji @FrancescoXX ? I love emoji but I dont know if you want to have them or not.

Let's see what Francesco thinks 😊

**Comment by FrancescoXX** [comment]
hey @JayrajSingh65 , I like the youtube section but the discord part I am not sure, I will tag the maintainer @ewainy

**Comment by ewainy** [comment]
What I really like about the discord suggestion is that it isn't a direct invite link, you can see more information about the group and then decide to join or not. I think this is a good one to open a discussion with and see what others think. I like it, but we need guidelines.

**Review by ewainy — CHANGES_REQUESTED** [review]
Please can you format Youtube tutorials section

I have opened a discussion for discord servers

File: README.md
Comment (ewainy) [inline README.md:218]: Capitalize:
## &#127916; YouTube Tutorials

I think next bit should be on a different line underneath heading, I have changed it a little, what do you think of this?

Explore and learn about crypto, blockchain, DAOs, EVM and more...

File: README.md
Comment (ewainy) [inline README.md:220]: This sends me to Gala Games?

File: README.md
Comment (ewainy) [inline README.md:222]: Can we capitalize:
Moralis Web3

File: README.md
Comment (ewainy) [inline README.md:223]: Capitalize:
Clever Programmer

File: README.md
Comment (ewainy) [inline README.md:219]: Capitalize: University

File: README.md
Comment (ewainy) [inline README.md:221]: Just Ethereum should do?

File: README.md
Comment (ewainy) [inline README.md:229]: You have linked to Peeps Platform again

Comment (FrancescoXX) [reply README.md:231]: Please take a look @JayrajSingh65 .

You can also remove the discord server section because the yt one looks great.

**Review by FrancescoXX — CHANGES_REQUESTED** [review]

**Review by Shukla007 — COMMENTED** [review]
Also, remove the extra space from the links for better link alignment.

**Comment by JayrajSingh65** [comment]
Ok i will do

File: README.md
Comment (JayrajSingh65) [reply README.md]: hey,  I removed it hope now its looking cool check it out

**Review by JayrajSingh65 — COMMENTED** [review]
I removed all the issues hope you guys like it

File: README.md
Comment (ewainy) [inline README.md:223]: This link still takes me to Gala Games so I am confused, Im not sure that's the right link as I'm expecting to see Whiteboard Crypto channel?
Screenshot 2022-08-16 at 15 49 23

**Review by ewainy — CHANGES_REQUESTED** [review]
This is looking great, thank you for taking all the feedback and revising!
One last thing, I have mentioned in the past, please could you look at the link for Whiteboard Crypto, it goes to Gala Games channel? Do you have the correct link to attach?

File: README.md
Comment (JayrajSingh65) [reply README.md:223]: Ya just typo mistake #solving

Comment (JayrajSingh65) [reply README.md]: I solved this issue please check it out I think everything is fine now.

**Review by ewainy — APPROVED** [review]
Brilliant, I'm happy with this now. I think this is ready to be merged, we will get Francesco to review the PR and merge - usually on a live stream.

**Comment by JayrajSingh65** [comment]
> Brilliant, I'm happy with this now. I think this is ready to be merged, we will get Francesco to review the PR and merge - usually on a live stream. 

I am super excited 😊 can't wait for live stream.

**Comment by FrancescoXX** [comment]
finally this merge happened!

**Pull request merged**
